packages/app/plugin/src/ios/appDelegate.ts

```diff
@@ -0,0 +1,48 @@
+import { ConfigPlugin, IOSConfig, withDangerousMod } from '@expo/config-plugins';
+import fs from 'fs/promises';
+
+const methodInvocationBlock = `[FIRApp configure];`;
+
+export function modifyObjcAppDelegate(contents: string): string {
+  // Add import
+  if (!contents.includes('@import Firebase;')) {
+    contents = contents.replace(
+      /#import "AppDelegate.h"/g,
+      `#import "AppDelegate.h"
+@import Firebase;`,
```

Comment:
Maybe wrap this in an optional in case someone failed to link the native module. The JS interface would produce a better error in JS then the Xcode build would.